mkosi/initrd.py

```diff
@@ -191,6 +191,16 @@ def add_raid_config() -> list[str]:
     return cmdline
 
 
+def add_vconsole_config() -> list[str]:
+    cmdline = []
+
+    for f in ("/etc/default/keyboard", "/etc/vconsole.conf"):
+        if Path(f).exists():
+            cmdline += ["--extra-tree", f"{f}:{f}"]
+
+    return cmdline
```

Comment:
What about this:
```
def vconsole_config() -> Iterator[str]:
     for f in ("/etc/default/keyboard", "/etc/vconsole.conf"):  
            if Path(f).exists():
                yield from ["--extra-tree", f"{f}:{f}"]
```
and then
```
cmdline += list(vconsole_config())
```

Reply:
The call to list is actually not needed, `list += iterator` works, making this a bit prettier.

But staying with a plain list instead of an iterator would be clearer, in my opinion.
```python
def vconsole_config() -> list[str]:
     return [
        f"--extra-tree={f}:{f}"
        for f in ("/etc/default/keyboard", "/etc/vconsole.conf")
        if Path(f).exists()]
     ]
```

Reply:
It makes sense, changed.